stats mir
stats arm
stats ainode
stats air_supply
stats paint_gun
stats realsense_camera
bool connectivity
bool arm_lock_flag
bool emergency_flag 
string current_process
int32 completion_percentage
int32 mir_battery
int32 ccs_battery
string gps_data
string temperature
string humidity
string tape_color































